src/Client.cpp

```diff
@@ -150,6 +150,13 @@ err_code_t Client::version(types::broadcast_result_t** results, size_t* nHosts)
 }
 
 
+err_code_t Client::quit() {
+  broadcastCommand(keywords::kQUIT, 4);
```

Comment:
Technically, relying on the compiler optimizations is not always a good practice.

You should take advantages of C++ templates to get the length of string literals during compile time.

```cpp
template <typename _CharType, size_t _Length>
inline size_t get_string_length(const _CharType (&)[_Length])
{
  return _Length - 1;
}
```